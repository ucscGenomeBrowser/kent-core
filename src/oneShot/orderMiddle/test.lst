f
b
c
d
e
a
