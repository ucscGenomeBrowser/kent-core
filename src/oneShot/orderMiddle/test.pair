a	b	1.1
a	c	2.1
a	d	3.1
a	e	4.1
a	f	5.1
b	c	1.2
b	d	2.2
b	e	3.2
b	f	4.2
c	d	1.3
c	e	2.3
c	f	3.3
d	e	1.4
d	f	2.4
e	f	1.5
