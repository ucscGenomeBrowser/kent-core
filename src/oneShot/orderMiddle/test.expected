f
e
d
c
b
a
